src/datum_api.c

```diff
@@ -439,11 +439,11 @@ bool datum_api_check_admin_password_only(struct MHD_Connection * const connectio
 	return false;
 }
 
-static enum MHD_DigestAuthAlgorithm datum_api_pick_digest_algo(struct MHD_Connection * const connection, const bool nonce_is_stale) {
+static enum MHD_DigestAuthAlgorithm datum_api_pick_digest_algo(struct MHD_Connection * const connection) {
 	const char * const ua = MHD_lookup_connection_value(connection, MHD_HEADER_KIND, "User-Agent");
 	if (strstr(ua, "AppleWebKit/") && !(strstr(ua, "Chrome/") || strstr(ua, "Brave/") || strstr(ua, "Edge/"))) {
 		static bool safari_warned = false;
-		if (!(nonce_is_stale && safari_warned)) {
+		if (!(safari_warned)) {
```

Comment:
Why this change? The current behaviour intends to log the message every time a login is attempted from Safari (just not "nonce changed" updates).